.travis.yml

```
@@ -16,9 +16,6 @@ matrix:
     - language: node_js
       node_js: 
         - "stable"
-      cache:
-        directories:
-          - frontend/node_modules
       before_script:
         - cd frontend && npm install
       script:
```

```suggestion
      after_script:
          - CI=false && npm run build 
      script:
```
I suggest you run `npm run build` in test stage so we could resolve potential npm errors outside of master